website/docs/advanced-topics/server-side-rendering.md

```diff
@@ -126,11 +126,77 @@ suspended.
 With this approach, developers can build a client-agnostic, SSR ready
 application with data fetching with very little effort.
 
+## SSR Hydration
+
+Hydration is the process that connects a Yew application to the
+server-side generated HTML file. By default, `ServerRender` prints
+hydratable string which includes additional information to facilitate hydration.
+When the `Renderer::hydrate` method is called, instead of start rendering from
+scratch, Yew will reconcliate the Virtual DOM generated by the application to
+the html string generated by the server renderer.
+
+:::caution
+
+To successfully hydrate an html representation created by the
+`ServerRenderer`, the client must produce a Virtual DOM layout that
+exactly matches the one used for SSR including components that does not
+contain any elements. If you have any component that is only useful in
+one implementation, you may want to use a `PhantomComponent` to fill the
+position of the extra component.
+:::
+
+## Component Lifecycle during hydration
+
+During Hydration, components are created in a different order
+(top-to-bottom for siblings). After a component is created, it will be
```

Comment:
Is the exact order important? I don't think it should be documented in which order components are called, this is not behaviour to rely on, not even for normal components, much less during hydration. The reconciler should reserve the right to render them in any order it sees fit.